schema.sql

```diff
@@ -0,0 +1,46 @@
+CREATE TABLE gener (
+  id INT PRIMARY KEY NOT NULL,
+  name VARCHAR(100)
+);
+
+CREATE TABLE author (
+  id INT PRIMARY KEY NOT NULL,
+  first_name VARCHAR(50),
+  last_name VARCHAR(50)
+);
+
+CREATE TABLE label (
+  id INT PRIMARY KEY NOT NULL,
+  title VARCHAR(200),
+  color VARCHAR(20)
+);
+
+CREATE TABLE item (
+  id INT PRIMARY KEY NOT NULL,
+  gener_id INT,
+  author_id INT,
+  lable_id INT,
+  publish_date DATE,
+  archived BOOLEAN,
+  CONSTRAINT gener_id
+    FOREIGN KEY (gener_id)
+      REFERENCES gener(id),
+ CONSTRAINT author_id
+    FOREIGN KEY (author_id)
+      REFERENCES author(id),
+  CONSTRAINT lable_id
+    FOREIGN KEY (lable_id)
+      REFERENCES label(id)
+);
+
+
+CREATE TABLE book (
```

Comment:
It's asked to follow the requirements as it mentions:

-  books table (add all properties and associations from the parent Item class as table columns)